src/components/User.jsx

```diff
@@ -0,0 +1,36 @@
+// Libraries
+import React, { useEffect, useState } from "react";
+import { API, graphqlOperation } from "aws-amplify";
+import { listUsers } from "../graphql/queries";
+
+const User = ({ children }) => {
+  const [userData, setUserData] = useState();
+
+  useEffect(() => {
+    fetchUser();
+  }, []);
+
+  const fetchUser = async () => {
+    try {
+      const userId = localStorage.getItem("user_id");
+
+      // Fetching all users from our database
+      const userData = await API.graphql(graphqlOperation(listUsers));
+
+      // Filtering users based on the twitter id and setting is as the data
+      setUserData(
+        userData.data.listUsers.items.filter(
+          (user) => user.twitterId === userId
+        )
+      );
+
+      console.log(userData.data.listUsers.items);
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  return <>{userData ? children(userData) : children}</>;
```

Comment:
Did you mean to add `<div>`?

Reply:
I don't want to create a DOM node, so it's not really needed. A [fragment](https://reactjs.org/docs/fragments.html) seemed like a better fit.